pkg/util/shared_action_queue_test.go

```diff
@@ -0,0 +1,147 @@
+package sync_test
```

Comment:
I think you need to rename this to `util_test` to keep Gazelle happy.

Reply:
Done.

Reply:
Done.